docs/binary.md

```diff
@@ -39,15 +48,15 @@ It is just like counting in decimal except we reach 10 much sooner.
 | 1            | Then 1                 |
 | 2-8          | Count 1,2,3,4,5,6,7,8  |
 | 9            | This is the **last digit** in Decimal|
-|**1**0        | So we start back at **0** again, but add **1** on the left|
+|**1**0        | So we start back at **0** again, but carry **1** on the left|
```

Comment:
Make this `but carry 1 on the left` bold as done in other tables to make it consistent

Reply:
done